### ISSUE TITLE
conditional age-at-length data

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [pacific-hake/hake-assessment#640]
Lengths are available but currently are not being used in the stock assessment because we don't estimate growth and we utilize age-based selectivity. I am uncertain if you added conditional-age-at-length data to the assessment if it could be informative given that there is no age-length key. It would be a good exercise to try, especially given that samplers have zero incentive to continue to collect length data when they perceive it as not being used. Sample sizes of lengths have decreased over time because of this perception.

**Comment by cgrandin** [comment]
Before 2011 those data were used in the assessment. In the 2011 assessment we saw that there were pathological patterns in the length data so the StAR panel and assessors agreed to do away with them in future assessments and go with age-based only.

**Comment by aaronmberger-nwfsc** [comment]
Yeah, we don't want to go back to estimating growth.  I think Kelli is interested in just looking at the length data to inform age-based selectivity (not growth) in conjunction with t.v. selectivity.  Given all the data issues/resolutions we have had recently, it still would be worth a look.  Next year though for sure...

**Comment by kellijohnson-NOAA** [comment]
@aaronmberger-nwfsc suggested fitting data as ghost data to see the fits but not need to actually include them in the likelihood as a logical first step.
